Sources/Models/Avatar.swift

```diff
@@ -35,13 +35,13 @@ public struct Avatar {
     /// The placeholder initials to be used in the case where no image is provided.
     ///
     /// The default value of this property is "?".
-    public var initals: String = "?"
+    public var initials: String = "?"
```

Comment:
This is breaking can we add a CHANGELOG entry for this under `Fixed`? You can follow the same pattern as our existing changelog. Also, since there are no existing entries for the next release -- I'll mention they just go under the `Upcoming Release` section.